Makefile.am

```diff
@@ -194,6 +196,41 @@ cc_oci_runtime_CFLAGS = \
 	$(LIBMNL_CFLAGS) \
 	$(UUID_CFLAGS)
 
+libexec_SCRIPTS = cc-proxy
+
+AM_V_GO    = $(am__v_GO_@AM_V@)
+am__v_GO_  = $(am__v_GO_@AM_DEFAULT_V@)
+am__v_GO_0 = @echo "  GOBUILD " $@;
+am__v_GO_1 =
+
+systemdservice_in_files =		\
+	proxy/cc-proxy.service.in	\
+	proxy/cc-proxy.socket.in	\
+	$(NULL)
+
+systemdservice_files = 		\
+	proxy/cc-proxy.service	\
+	proxy/cc-proxy.socket	\
+	$(NULL)
```

Comment:
Hum, not sure I follow you, why would I create symlinks? The above installs the units file into $(systemdsystemunitdir)